conda_smithy/templates/README.md.tmpl

```diff
@@ -2,30 +2,40 @@
 {#-
 # -*- mode: jinja -*-
 -#}
-{%- set license_url = package_about.license_url -%}
 
-About {{ package_name }}
-======{{ '=' * package_name|length }}
+Feedstock license: [BSD-3-Clause](https://github.com/{{ github.user_or_org }}/{{ github.repo_name }}/blob/{{ github.branch_name }}/LICENSE.txt)
```

Comment:
Maybe add a `About {{ package_name }}-feedstock` at the top?

And then the other About sections are subheaders.

Reply:
Fixed in 5f76053.